packages/core/src/components/use-component-style.tsx

```diff
@@ -86,8 +86,9 @@ const getResponsiveStyles =
       )({ isMulti })
     } else {
       const { queries = [] } = props.theme.__breakpoints ?? {}
-      const { breakpoint } = props.theme.__config ?? {}
-      const isDown = breakpoint === "down"
+      const { direction = "down", identifier = "@media screen" } =
```

Comment:
[nits]
@hirotomoyamada 
Don't you need to assign to this identifier here?
Because the default argument is passed [here](https://github.com/yamada-ui/yamada-ui/pull/2118/files#r1644639914).

Reply:
Fixed